scripts/github-status/github-status.rb

```diff
@@ -48,6 +49,11 @@ def get_sha_status(sha)
     get_full_sha_status(sha).state rescue ''
   end
 
+  def get_pr_info(pr)
+    pr_stat = @client.pull_request(@repository, pr)
+    pr_stat.to_attrs
```

Comment:
thats not very shell-scriptable. I would prefer to be able to query a specific key from the json blob, and it would just return the value of that one. e.g.

-a get-pr-info -p <prid> -k base.head.owner

(where base.head.owner is the dotted path of the json key)